unison-cli/src/Unison/CommandLine.hs

```diff
@@ -68,7 +68,7 @@ watchConfig path =
       (config, t) <- autoReload autoConfig [Optional path]
       pure (config, killThread t)
 
-watchFileSystem :: TQueue Event -> FilePath -> IO (IO ())
+watchFileSystem :: Q.TQueue Event -> FilePath -> IO (IO ())
```

Comment:
Hehe, why delete the unqualified TQueue import and rewrite the TQueues as Q.TQueue? To have one less import line I suppose?

Reply:
I was getting "ambiguous TQueue" because I added the STM import; TBH I didn't think too hard about it haha.